src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/MobilePreviewEditModePageObject.java

```diff
@@ -0,0 +1,107 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.article.editmode;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class MobilePreviewEditModePageObject extends BasePageObject {
+
+  private static final String articleContentSelector = ".article-content p";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)